src/CMakeLists.txt

```diff
@@ -181,7 +190,11 @@ DLAF_addSublibrary(
           matrix_mirror.cpp
           memory/memory_view.cpp
           memory/memory_chunk.cpp
-  GPU_SOURCES cusolver/assert_info.cu cusolver/stedc.cu lapack/gpu/lacpy.cu lapack/gpu/laset.cu
+  GPU_SOURCES
+    cusolver/assert_info.cu
+    cusolver/stedc.cu
+    lapack/gpu/lacpy.cu
+    lapack/gpu/laset.cu
```

Comment:
The different handling of `SOURCES` and `GPU_SOURCES` is terrible. Any clue on why this is happening?

Reply:
I didn't even notice how bad that was. It seems to be controlled by https://github.com/eth-cscs/DLA-Future/blob/c071856a6373b7b6a15b0a90d7e036ed109fb234/.cmake-format.py#L117. `GPU_SOURCES` is 11 characters long and forces the other layout.